### PR TITLE
feat: add EUPHORIA chain config and tAURA warp token

### PR DESCRIPTION
**Pull request by hoanm**
### Description

- Register information of Euphoria Testnet
- Added warp information of tAURA token from Holesky to Euphoria 

### Backward compatibility

Yes

### Testing

Manually tested using Hyperlane CLI

### CODE REVIEW

File: chains/euphoria/logo.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+<defs>
+<linearGradient id="paint0_linear_1178_6223" x1="624.933" y1="768.413" x2="373.08" y2="162.818" gradientUnits="userSpaceOnUse">
+<stop stop-color="#FFD569" style="stop-color:#FFD569;stop-color:color(display-p3 1.0000 0.8353 0.4118);stop-opacity:1;"/>
+<stop offset="0.515056" stop-color="#FC8E4F" style="stop-color:#FC8E4F;stop-color:color(display-p3 0.9882 0.5569 0.3098);stop-opacity:1;"/>
+<stop offset="1" stop-color="#FF697B" style="stop-color:#FF697B;stop-color:color(display-p3 1.0000 0.4118 0.4824);stop-opacity:1;"/>
+</linearGradient>
+</defs>
+</svg>
```
Comment (paulbalaji) [inline chains/euphoria/logo.svg:10]: ran through svg minifier https://jakearchibald.github.io/svgomg/
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="898" height="973" fill="none"><path fill="url(#a)" d="M835.6 570a575 575 0 0 1-18.6-11 85.6 85.6 0 0 1 0-145.8 476 476 0 0 1 18.6-11.1A120.3 120.3 0 0 0 881 236.2c-34.1-58-109.7-78-168.7-44.4a414 414 0 0 1-9.2 5c-58.3 31.7-130.7-8.8-130.8-74.2v-1.2C572.2 54.4 516.9 0 448.7 0a122.5 122.5 0 0 0-123.4 121.4v1.2c-.2 65.5-72.6 105.9-130.9 74.3l-9.2-5.1a124.6 124.6 0 0 0-168.6 44.4c-34.1 58.1-14 132.4 45.2 165.9 6.3 3.6 12.5 7.3 18.6 11a85.6 85.6 0 0 1 0 145.9 476 476 0 0 1-18.6 11 120.3 120.3 0 0 0-45.2 166c34 58 109.6 77.9 168.6 44.4l9.2-5c58.3-31.7 130.7 8.7 130.9 74.1v1.3c0 67 55.2 121.4 123.4 121.4s123.5-54.4 123.5-121.4v-1.3c.1-65.4 72.5-105.8 130.8-74.3l9.2 5.1A124.6 124.6 0 0 0 880.9 736c34-58.1 13.9-132.4-45.3-165.9Zm-7 136.2a63.5 63.5 0 0 1-86.2 22.7 596 596 0 0 0-587.4 0A63.6 63.6 0 0 1 69 706.2a61.5 61.5 0 0 1 23-84.7C273.2 518.7 385.7 327 385.7 121.4c0-34.2 28.4-62 63.1-62 34.8 0 63 27.8 63 62 0 205.6 112.6 397.3 293.8 500.1a61.9 61.9 0 0 1 29.3 37.7c4.4 16 2.2 32.7-6.3 47Z"/><defs><linearGradient id="a" x1="624.9" x2="373.1" y1="768.4" y2="162.8" gradientUnits="userSpaceOnUse"><stop stop-color="#FFD569" style="stop-color:#ffd569;stop-color:color(display-p3 1 .8353 .4118);stop-opacity:1"/><stop offset=".5" stop-color="#FC8E4F" style="stop-color:#fc8e4f;stop-color:color(display-p3 .9882 .5569 .3098);stop-opacity:1"/><stop offset="1" stop-color="#FF697B" style="stop-color:#ff697b;stop-color:color(display-p3 1 .4118 .4824);stop-opacity:1"/></linearGradient></defs></svg>
```

File: chains/euphoria/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://euphoria-api.aurascan.io/
```
Comment (paulbalaji) [inline chains/euphoria/metadata.yaml:3]:
```suggestion
  - apiUrl: https://euphoria-api.aurascan.io
```

File: chains/euphoria/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    family: other
+    name: AuraScan
+    url: https://euphoria.aurascan.io/
```
Comment (paulbalaji) [inline chains/euphoria/metadata.yaml:6]:
```suggestion
    url: https://euphoria.aurascan.io
```

File: chains/euphoria/addresses.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+domainRoutingIsmFactory: "0xeFf432cF6Af184E62C43b43dBc86A7C79a66bC40"
```
Comment (paulbalaji) [inline chains/euphoria/addresses.yaml:1]: could you add `merkleTreeHook` and `interchainGasPaymaster` addresses as well please? you should be able to find these addresses by looking at the deployment logs

Comment (hoanm) [reply chains/euphoria/addresses.yaml:1]: Thanks for your comments. I found `merkleTreeHook` in the ./config/core-config.yaml but the `interchainGasPaymaster` is still missing. Where can I find it?

**Comment by paulbalaji** [comment]
please add a minor changeset using `yarn changeset add`

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**